soc/arm64/nxp_imx/imx8m/soc.h

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
soc.h is gone for arm64, please remove it.

Reply:
Hi, @gmarull , thanks for your comments, but there are still some drivers need soc.h, such as:
drivers/clock_control/clock_control_mcux_ccm.c
drivers/pinctrl/pinctrl_imx.c
drivers/serial/uart_mcux_iuart.c

So how to handle this issue? just create a dumy one? thanks.

Reply:
Hi, @gmarull , thanks for your comments, but there are still some drivers need soc.h, such as:
drivers/clock_control/clock_control_mcux_ccm.c
drivers/pinctrl/pinctrl_imx.c
drivers/serial/uart_mcux_iuart.c

So how to handle this issue? just create a dumy one? thanks.

Reply:
then please fix drivers to include the right headers.

Reply:
okay, I removed soc.h from this PR, and created another PR to fix drivers:
https://github.com/zephyrproject-rtos/zephyr/pull/49734

Thanks.